GitHub-Repositories-Repository/github-repositories-repository.json

```diff
@@ -245,6 +249,7 @@
     "Name"
   ],
   "readOnlyProperties": [
+    "/properties/Id",
```

Comment:
Might break existing stack updates. Please try testing following scenario : 
1.Create a stack with out this changes(using current prod changes).
2.Update the stack with this changes.